src/main/java/cofh/thermaldynamics/block/BlockDuct.java

```diff
@@ -282,16 +282,21 @@ public ItemStack getPickBlock(IBlockState state, RayTraceResult target, World wo
 		if (target.subHit >= 14 && target.subHit < 20) {
 			TileGrid tileEntity = (TileGrid) world.getTileEntity(pos);
 			Attachment attachment = tileEntity.getAttachment(target.subHit - 14);
-			ItemStack pickBlock = attachment.getPickBlock();
-			if (pickBlock != null) {
-				return pickBlock;
+			if(attachment != null) {
+				ItemStack pickBlock = attachment.getPickBlock();
+				if (pickBlock != null) {
+					return pickBlock;
+				}
 			}
 		}
 		if (target.subHit >= 20 && target.subHit < 26) {
 			TileGrid tileEntity = (TileGrid) world.getTileEntity(pos);
-			ItemStack pickBlock = tileEntity.getCover(target.subHit - 20).getPickBlock();
```

Comment:
context: same as for the attachment; calling `getPickBlock()` on the cover would cause a NPE directly after dismantling it with a crescent hammer